tests/test_slow_retrieval_attack.py

```diff
@@ -234,6 +210,13 @@ def tearDown(self):
     tuf.roledb.clear_roledb(clear_all=True)
     tuf.keydb.clear_keydb(clear_all=True)
```

Comment:
these tuf calls should not be needed if this function just wipes everything anyway